openapi-spec/swagger.yaml

```diff
@@ -36,34 +46,34 @@ tags:
     description: Muli-cloud cloud storage backends
     externalDocs:
       description: Find out more
-      url: 'https://opensds.io'
-  - name: type
+      url: 'https://sodafoundation.io/'
+  - name: types
     description: Supported storage backends
     externalDocs:
       description: Find out more
-      url: 'https://opensds.io'
+      url: 'https://sodafoundation.io/'
   - name: plan
     description: Migration plan
     externalDocs:
       description: Find out more
-      url: 'https://opensds.io'
+      url: 'https://sodafoundation.io/'
   - name: job
     description: Migration job
     externalDocs:
       description: Find out more
-      url: 'https://opensds.io'
+      url: 'https://sodafoundation.io/'
   - name: policy
     description: Schedule policy of plan.
     externalDocs:
       description: Find out more
-      url: 'https://opensds.io'
+      url: 'https://sodafoundation.io/'
   - name: s3
     description: s3 interface of multi-cloud
     externalDocs:
       description: Find out more
-      url: 'https://opensds.io'
+      url: 'https://sodafoundation.io/'
 paths:
-  '/v1/{tenantId}/backends':
+  '/{tenantId}/backends':
```

Comment:
How did we remove v1 from the backend url?

Reply:
@sfzeng 
I think the changes were made by you only?
You can check the following:
https://github.com/sodafoundation/multi-cloud/commit/a961bcc94530afb7a67615f3adf21448972c3a42#diff-2baa3cfcc6a4fa749254f0d9510c72d4L45
![image](https://user-images.githubusercontent.com/8712951/83988924-608d8d00-a962-11ea-8157-3ee2cad81757.png)


The swagger changes are according to current working code and API changes.

Reply:
Only s3 APIs are changed in this PR, backends are not s3 APIs. Backends related API are soda specific to support multi-clouds backend. 
If you check the normal S3 API, you will see GET /tenantid/backends means _get an object which name is bakcends_ from the _bucket which name is tenantid_.

Reply:
If is possible, plz make a simple test for each changed API. Thanks.

Reply:
@sfzeng 
With respect to your commit.
https://github.com/sodafoundation/multi-cloud/commit/a961bcc94530afb7a67615f3adf21448972c3a42
```
1. Seperate s3 APIs from other APIs and make it's URL be compatible with aws s3. 2. Modificate authentication algorithm and make it support auth v2/v4.
 master (#903)  v0.12.0
@sfzeng
sfzeng committed on Mar 13 
1 parent 63a5421 commit a961bcc94530afb7a67615f3adf21448972c3a42
```
PR link:
https://github.com/sodafoundation/multi-cloud/pull/903

https://github.com/sodafoundation/multi-cloud/commit/a961bcc94530afb7a67615f3adf21448972c3a42#diff-2baa3cfcc6a4fa749254f0d9510c72d4L45


https://github.com/sodafoundation/multi-cloud/commit/a961bcc94530afb7a67615f3adf21448972c3a42#diff-2baa3cfcc6a4fa749254f0d9510c72d4R62

Line 45 ws.Path("/v1")
is changed to
Line 62 ws.Path("/")

I just wanted to know How was the testing done for the above commit/PR. I will follow the same for this PR.
If there was only s3-compatible changes done, then why the others API got effected and without any release note.

I am just following the currently available API code, nothing special.
You can refer the following code.
https://github.com/sodafoundation/multi-cloud/blob/master/api/cmd/main.go#L62